### PR TITLE
DP-1.8 - Two strict priority queue traffic test - Adding required vendor specific queue name change and interface-ref definition

### PR DESCRIPTION
**Pull request by charantejag504**
DP-1.8 - Two strict priority queue traffic test - Adding required vendor specific queue name change and interface-ref definition
1.   Added Queue ID as required by Juniper.
     Other pull request containing such changes: https://github.com/openconfig/featureprofiles/pull/1087
     Related OC model change: https://github.com/openconfig/public/pull/772
2.  Included existing deviation  ExplicitInterfaceRefDefinition for interface configuration under QoS(classifier and scheduler)

### CODE REVIEW

File: feature/qos/ate_tests/two_sp_queue_traffic_test/two_sp_queue_traffic_test.go [hunk 1]
```diff
@@ -1197,38 +1197,65 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
...
+		be0: "BE0",
+		be1: "BE1",
+		af1: "AF1",
+		af2: "AF2",
+		af3: "AF3",
+		af4: "AF4",
+		nc1: "NC1",
+	}
+
+	if dut.Vendor() == ondatra.JUNIPER {
```
Comment (greg-dennis) [inline feature/qos/ate_tests/two_sp_queue_traffic_test/two_sp_queue_traffic_test.go:1215]: Where is the deviation that tracks the need for this change?

Comment (charantejag504) [reply feature/qos/ate_tests/two_sp_queue_traffic_test/two_sp_queue_traffic_test.go:1215]: This is not a deviation. it is a vendor specific queue name, No test expectation is modified. it is already approved 
https://github.com/openconfig/featureprofiles/blob/main/feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go

Comment (greg-dennis) [reply feature/qos/ate_tests/two_sp_queue_traffic_test/two_sp_queue_traffic_test.go:1215]: Ok, I got confirmation that we are going to punt on this non-uniformity for now. I wish an issue had been created to arrive at a more uniform solution (such a helper naming library), but it hasn't. I'll track it elsewhere internally.

**Review by greg-dennis — APPROVED** [review]
Approving for Go quality only

**Review by greg-dennis — APPROVED** [review]
Re-approving for Go quality only

**Review by sachendras — APPROVED** [review]

**Review by rszarecki — APPROVED** [review]

**Pull request merged**